client/my-sites/google-my-business/controller/index.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * External Dependencies
+ *
+ * @format
+ */
+
```

Comment:
There is an unnecessary blank line here. Also shouldn't the `@format` comment in its own docblock like in other files?

```javascript
/** @format */

/**
 * External dependencies
 */
```